README.md

```diff
@@ -31,152 +42,177 @@ Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) d
+- Select the relevant SDK for your target. (Publisher SDK, Subscriber SDK or both)
+- [This apple guide](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) explains the steps in more detail.
 - To install this package into a **Swift Package**, add the following to your manifest (`Package.swift`):
 
-  ```swift
-  .package(url: "https://github.com/ably/ably-asset-tracking-swift", from: LATEST_VERSION),
-  ```
-    - You can find a version in the [releases](https://github.com/ably/ably-asset-tracking-swift/releases) page. 
+```swift
+
+.package(url: "https://github.com/ably/ably-asset-tracking-swift", from: LATEST_VERSION),
+
+```
+
+_You can find the version on the [releases](https://github.com/ably/ably-asset-tracking-swift/releases) page._
+
+## Example Apps
+
+- Configure your Mapbox credentials (`~/.netrc`) to download the Mapbox SDK by following [this](https://docs.mapbox.com/ios/search/guides/install/#configure-credentials) guide. You'll need a Mapbox account.
+- An `Examples/Secrets.xcconfig` file containing credentials (keys/ tokens) is required to build the example apps. (You can use the example `Examples/Example.Secrets.xcconfig`, e.g. by running `cp Examples/Example.Secrets.xcconfig Examples/Secrets.xcconfig`). Update the following values in `Examples/Secrets.xcconfig`:
+- `ABLY_API_KEY`: Used by all example apps to authenticate with Ably using basic authentication. Not recommended in production, and can be taken from [here](https://ably.com/accounts).
+- `MAPBOX_ACCESS_TOKEN`: Used to access Mapbox Navigation SDK/ APIs, and can be taken from [here](https://account.mapbox.com/). Using the Mapbox token is only required to run the **Publisher** example apps.
+- Open `AblyAssetTracking.xcworkspace` to open an Xcode workspace containing example apps and the Swift Package containing the SDKs that showcase how to use the Ably Asset Tracking SDKs.
 
 ## Usage
 
 ### Publisher SDK
 
-The Asset Publisher SDK can be used to efficiently acquire the location data on a device, and publish location updates to other subscribers in real time. Here is an example of how the Asset Publisher SDK can be used:
+The Asset Publisher SDK can efficiently acquire the location data on a device and publish location updates to other subscribers in realtime. Here is an example of how the Asset Publisher SDK can be used:
 
 ```swift
+
 import AblyAssetTrackingPublisher
 
 // Initialise a Publisher
+
 publisher = try? PublisherFactory.publishers() // get a Publisher Builder
-  .connection(ConnectionConfiguration(apiKey: ABLY_API_KEY,
-                                      clientId: CLIENT_ID)) // provide Ably configuration with credentials
-  .log(LogConfiguration()) // provide logging configuration
-  .transportationMode(TransportationMode()) // provide mode of transportation for better location enhancements
-  .delegate(self) // provide delegate to handle location updates locally if needed
-  .start()
+
```

Comment:
I don't like the vertical spacing (blank lines) between each line here. It looks odd [when rendered](https://github.com/ably/ably-asset-tracking-swift/tree/feature/create-contributing-update-readme-md#publisher-sdk). Same for the subscriber example. We don't do this elsewhere, e.g. [here, for Android](https://github.com/ably/ably-asset-tracking-android#publishing-sdk).

Reply:
fixed here 8d9502ef2d7a61158697bf9ea7e5602796959f65